src/main/java/com/gtnewhorizons/angelica/mixins/early/angelica/misc/MixinEntityPlayerSP_DynamicFOV.java

```diff
@@ -0,0 +1,20 @@
+package com.gtnewhorizons.angelica.mixins.early.angelica.misc;
+
+
+import jss.notfine.core.Settings;
+import net.minecraft.client.entity.EntityPlayerSP;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(EntityPlayerSP.class)
+public class MixinEntityPlayerSP_DynamicFOV {
+
+    @Inject(method = "getFOVMultiplier", at = @At("RETURN"), cancellable = true)
```

Comment:
Please document why we're doing a mixin instead of using the event

Reply:
Changed back to event based